app/views/spree/shared/_paypal_checkout_button.html.erb

```diff
@@ -15,7 +15,9 @@
   }
 
   var paypalOptions = {
-    flow: 'vault',
+    flow: '<%= SolidusPaypalBraintree::Gateway.first.preferred_paypal_flow %>',
```

Comment:
@tvdeyen no, because you might use the Vault flow without storing the payment method in the vault.

Reply:
The whole purpose of the `vault` flow is to store payment methods for later re-use, no? Why should one use the `vault` flow if the payment method must not be stored in it? Do you have an example of that use case?

Reply:
@tvdeyen I'm not sure about the use case, but there are two points in favor of keeping these separate in my opinion:

- We can't know the return value of `store_in_vault` _before_ we start the PayPal flow.
- PayPal supports using the Vault flow without storing the payment source, so I don't think we should prevent users from doing this, especially since it comes at no additional cost for us.